lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -117,6 +23,19 @@
     border-radius: 4px;
     color: #444;
   }
+
+  .slideshow__slide .container > .wrapper {
+    margin: 12px;
```

Comment:
Can we avoid using pixel widths here? It looks like the first `12px` is just `$base-spacing / 2`, but I'm pretty lost on the other ones. If they're based on our grid/spacing, use the appropriate mix-ins or variables. Otherwise, can you add a comment explaining the choice?

This should also be applied as a modifier on the `.container` pattern (like `.container.-compact` or something) rather than reaching in and mucking with it based on what element it's being held in).